bundles/org.openhab.binding.volumio/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,176 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="volumio"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="player">
+		<label>Volumio Binding Thing</label>
+		<description>A Volumio Instance</description>
+
+		<channels>
+			<channel id="title" typeId="title"/>
+			<channel id="artist" typeId="artist"/>
+			<channel id="album" typeId="album"/>
+			<channel id="volume" typeId="volume"/>
+			<channel id="player" typeId="player"/>
+			<channel id="track-type" typeId="track-type"/>
+			<channel id="play-radio-stream" typeId="play-radio-stream"/>
+			<channel id="playPlaylist" typeId="playPlaylist"/>
+			<channel id="clearQueue" typeId="clearQueue"/>
+			<channel id="playURI" typeId="playURI"/>
+			<channel id="playFile" typeId="playFile"/>
+			<channel id="random" typeId="playRandom"/>
+			<channel id="repeat" typeId="playRepeat"/>
+			<channel id="systemCommand" typeId="systemCommand"/>
+			<channel id="stopCommand" typeId="stopCommand"/>
```

Comment:
This should  follow the [naming conventions](https://next.openhab.org/docs/developer/guidelines.html#naming-convention).

Reply:
solved